common/src/main/java/com/wynntils/core/functions/arguments/FunctionArguments.java

```diff
@@ -151,6 +152,15 @@ public Double getDoubleValue() {
             return (Double) this.getValue();
         }
 
+        public CappedValue getCappedValue() {
+            if (this.type == Number.class) {
+                int val = ((Number) this.getValue()).intValue();
+                return new CappedValue(val, val);
+            }
```

Comment:
Should this ever happen? This is like checking if an int value is 0 or 1 in getBooleanValue, when the actual value is a string.